File: 1791. Find Center of Star Graph/findCenter.py
from typing import List

class Solution:
  def findCenter(self, edges: List[List[int]]) -> int:
    if edges[0][0] in edges[1]:
      return edges[0][0]
    return edges[0][1]


edges = [[1,2],[2,3],[4,2]]
solution = Solution()
print('findCenter: ', solution.findCenter(edges))

File: 1791. Find Center of Star Graph/findCenter.js
/**
 * @param {number[][]} edges
 * @return {number}
 */
const findCenter = (edges) => {
  if (edges[0][0] === edges[1][0] || edges[0][0] === edges[1][1]) {
    return edges[0][0];
  }

  return edges[0][1];
};


const edges = [[1, 2], [2, 3], [4, 2]];
console.log('findCenter: ', findCenter(edges));
